packages/SwingSet/test/gc-helpers.js

```diff
@@ -375,11 +371,12 @@ export function validateDeleteMetadataOnly(
   }
   let priorKey = '';
   if (entries >= 0) {
+    validate(v, matchVatstoreSet(`vc.${idx}.|entryCount`, `0`));
```

Comment:
I think this would go away if we avoided the "tiny regression" extra length=0 set, right?

Reply:
Yep.